cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorage.kt

```diff
@@ -35,6 +35,7 @@ internal interface DataStorage : DataStorageGdpr, DataStorageCcpa {
 
     var consentStatusResponse: String?
     var gdprConsentStatus: String?
+    val ccpaApplies: Boolean
```

Comment:
Why did you decide to move `ccpaApplies` from `DataStorageCcpa` class here?

Reply:
@andresilveirah because it is a "mirroring" parameter, and we decided to actually access it via `ccpaConsentStatus` from the DataStorageCcpa